script.js

```diff
@@ -0,0 +1,228 @@
+    if (!firstValue && !secondValue) {
+        return false;
+    }
+    return true;
+}
+
+console.log(isOnlyOneOfThemTruthy("Bye"));
+console.log(isEitherOneOfThemTruthy(0, 2));
+
+// 4.Create a function called _if which takes 3 arguments: a value bool and 2 functions (which do not take any parameters): func1 and func2
+// When bool is truthy, func1 should be called, otherwise call the func2.
+
+function firstFunction() {
+    console.log('True')
+}
+function secondFunction() {
+    console.log('False')
+}
+
+function isBoolTruthy(booleanValue, firstFunction, secondFunction) {
+    if (booleanValue) {
+        return firstFunction();
+    }
+    return secondFunction();
+}
+
+isBoolTruthy(true, firstFunction, secondFunction);
+
+// 5.After a hard quarter in the office you decide to get some rest on a vacation.
+// So you will book a flight for you and your girlfriend and try to leave all the mess behind you.
+// You will need a rental car in order for you to get around in your vacation.
+// The manager of the car rental makes you some good offers.
+// Every day you rent the car costs $40. If you rent the car for 7 or more days, you get $50 off your total.
+// Alternatively, if you rent the car for 3 or more days, you get $20 off your total.
+// Write a code that gives out the total amount for different days(d).
+
+function getTheTotalAmountForCarRental(dayNumber) {
+    const costPerDay = 40 * dayNumber;
+    const fiftyOff = 50;
+    const twentyOff = 20;
+    if (dayNumber >= 7) {
+        return costPerDay - fiftyOff;
+    }
+    if (dayNumber >= 3) {
+        return costPerDay - twentyOff;
+    }
+    return costPerDay;
+}
+
+console.log(getTheTotalAmountForCarRental(6));
+
+// 6. Americans are odd people: in their buildings, the first floor is actually the first floor and there is no 13th floor (due to superstition).
+// Write a function that given a floor in the american system returns the floor in the european system.
+// With the 1st floor being replaced by the first floor and the 13th floor being removed, the numbers move down to take their place. In case of above 13, they move down by two because there are two omitted numbers below them.
+// Basements (negatives) stay the same as the universal level.
+
+function getTheFloorInTheEuropeanSystem(americanFloorNumber) {
+    if (americanFloorNumber <= 0) {
+        return americanFloorNumber;
+    }
+    if (americanFloorNumber <= 12) {
+        return americanFloorNumber - 1;
+    }
+    if (americanFloorNumber >= 14) {
+        return americanFloorNumber - 2;
+    }
+}
+
+console.log(getTheFloorInTheEuropeanSystem(14));
+
+// 7. In the rock-paper-scissors.js file.
+
+// 8. Write the isValidPassword function.
+// Make sure that the password is at least 8 characters long and contains at least one number and one uppercase letter.
+
+function getLettersAmount(word) {
+    return word.length;
+}
+
+function getUppercaseLetter(word) {
```

Comment:
I would change this function to return `true` if there is an uppercase letter and `false` if there is none. You would also have to change the name of the function.